ballerina/client.bal

```diff
@@ -73,60 +100,47 @@ public isolated client class Client {
     # + return - Summary of the execution and results are returned in an `sql:ProcedureCallResult`, or an `sql:Error`
     remote isolated function call(sql:ParameterizedCallQuery sqlQuery, typedesc<record {}>[] rowTypes = [])
     returns sql:ProcedureCallResult|sql:Error = @java:Method {
-        'class: "io.ballerina.stdlib.java.jdbc.nativeimpl.CallProcessor",
+        'class: "io.ballerina.lib.snowflake.nativeimpl.CallProcessor",
         name: "nativeCall"
     } external;
 
     # Closes the SQL client and shuts down the connection pool.
     #
     # + return - Possible error when closing the client
     public isolated function close() returns sql:Error? = @java:Method {
-        'class: "io.ballerina.stdlib.java.jdbc.nativeimpl.ClientProcessor",
+        'class: "io.ballerina.lib.snowflake.nativeimpl.ClientProcessor",
         name: "close"
     } external;
 }
 
 # An additional set of configurations related to a database connection.
-#
-# + datasourceName - The driver class name to be used to get the connection
-# + properties - The database properties, which should be applied when getting the connection
-# + requestGeneratedKeys - The database operations for which auto-generated keys should be returned
 public type Options record {|
+    # The driver class name to be used to get the connection
```

Comment:
Shouldn't this added to a different file(types.bal)?

Reply:
We only have ClientConfiguration and Options records in Snowflake. Keeping them in the client.bal is more suitable than moving to a different file.